src/components/modals/password/password.module.css

```diff
@@ -0,0 +1,16 @@
+.password-modal {
+	padding-top: 137px;
+	backdrop-filter: blur(10px);
+}
+
+.password-modal :global .modal-content {
+	background-color: transparent;
```

Comment:
This needs to have some color.
![Screenshot 2022-06-01 at 18 36 48](https://user-images.githubusercontent.com/7974813/171455759-bcede57d-bb51-4661-9413-03cf5f97364d.png)